infra/discovery/src/apollo/README.md

```diff
@@ -1,38 +1,36 @@
-# Discovery Servers
+# Discovery Server
 
 The discovery server provides searching and content moderation features.
 
-### Configuration ENV variables
+## Configuration ENV variables
 
-MODERATOR_ADDRESSES - a comma sperated (no space), list of ETH addresses that can perform moderation actions.
+`MODERATOR_ADDRESSES` - a comma sperated (no space), list of ETH addresses that can perform moderation actions.
+
+## To start the Apollo GraphQL server
 
-# To start the Apollo GraphQL server
 You can start the listener using either docker compose or lerna.
 
-## Docker compose
+### Docker compose
+
 Start the origin-discovery container.
 
     docker-compose up origin-discovery
 
-## Lerna
-
-### Prerequisite
- - Setup Postgres locally.
- - Run migration scripts to create the schema
- ```
-lerna run migrate --scope origin-discovery
-```
- - Install Elasticsearch locally.
- - Create the Elasticsearch schema
- ```
- cd origin/src/origin-discovery/devops
- ./es-cli.js createIndex
- ```
-### Starting the server
-```
-export DATABASE_URL=postgres://origin:origin@localhost/origin
-export ELASTICSEARCH_HOST=localhost:9200
-lerna run start:discovery --scope origin-discovery --stream
-```
-
-The server should start and you can now point your browser to http://localhost:4000/graphql to access the GraphQL playground.
+### Lerna
+
+#### Prerequisite
+
+- Setup Postgres locally.
+- Run migration scripts to create the schema
+  `lerna run migrate --scope origin-discovery`
```

Comment:
I think it is --scope @origin/discovery
Also the identity, growth and bridge migrations need to get run as well
lerna run migrate --scope @origin/identity
lerna run migrate --scope @origin/growth
lerna run migrate --scope @origin/bridge

Reply:
Will do another pass of READMEs... I think a lot of them are out of date